flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/logical/FlinkLogicalDataStreamTableScan.scala

```diff
@@ -41,12 +40,13 @@ import java.util.function.Supplier
 class FlinkLogicalDataStreamTableScan(
     cluster: RelOptCluster,
     traitSet: RelTraitSet,
+    hints: util.List[RelHint],
     table: RelOptTable)
-  extends TableScan(cluster, traitSet, Collections.emptyList[RelHint](), table)
+  extends TableScan(cluster, traitSet, hints, table)
```

Comment:
Should we add hints in `explainTerms` for this node?

Reply:
yes, it's better for us to consider all the TableScans in this pr